packages/gestalt/src/SegmentedControl/SegmentedControl.js

```diff
@@ -36,14 +36,18 @@ export default function SegmentedControl(props: Props) {
             onClick={e => onChange({ event: e, activeIndex: i })}
             role="tab"
           >
-            <Text
-              bold
-              color={isSelected ? 'darkGray' : 'gray'}
-              align="center"
-              size={size}
-            >
-              {item}
-            </Text>
+            {typeof item === 'string' ? (
+              <Text
+                bold
+                color={isSelected ? 'darkGray' : 'gray'}
+                align="center"
+                size={size}
+              >
+                {item}
+              </Text>
+            ) : (
+              <div>{item}</div>
```

Comment:
@loserford could we add use `Box display="flex" justifyContent="center"` instead of this `div`? 
That way
 - We'll make sure that the content of each items is always centered
 - We can remove the wrapping `Box` in the example

![image](https://user-images.githubusercontent.com/127199/40818973-c2bd47e4-650e-11e8-8af7-248c0ec40eb2.png)

/cc @chrislloyd